spec/index.bs

```diff
@@ -7381,35 +7380,36 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
     - {{GPUProgrammableStage}} |descriptor|
     - {{GPUPipelineLayout}} |layout|
 
-    Return `true` if all requirements in the following steps are satisfied, and `false` otherwise:
+    All of the requirements in the following steps |must| be met.
```

Comment:
If we're going to start emphasizing `must` here, should we also consider using [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) `MUST` instead, as is the case for most other W3C standards?

Reply:
We do, there's just no capitalization or emphasis to indicate it. Bikeshed generates this section for us: https://gpuweb.github.io/gpuweb/#w3c-conventions

Reply:
Ah! I didn't remember this section! Thanks.